### PR TITLE
fix : "last_name" in create attendee api is now required

### LINKED ISSUE #4676: Should "Last name" in create attendee api be required?
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-server

**Current behavior:**
Docs say that the last name field in create an attendee is required but None is allowed in the API.

**Expected behavior:**
@iamareebjamal @SaptakS @niranjan94 @poush @shubham-padia What should be the expected behaviour here - required or not required?

**Steps to reproduce:**
Create an attendee with no last name field. It is successfully created.

### PR DESCRIPTION
**Pull request by mayank8318**
Fixes #4676 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
"last_name" field in create attendee api is a required field now, as stated in the docs.

#### Changes proposed in this pull request:

- Make "last_name" a required field since the docs state it as required.

### CODE REVIEW

**Comment by shubham-padia** [comment]
This might create problems when migrating the existing database as there maybe attendees without any last name. We need a migration to handle that. I'm not sure right now how we should go about that. Till we figure it out, we can make the field required in the API schema and let the db schema be as it was before. A follow-up issue can be made to handle change in the db schema. @poush @niranjan94 thoughts?

**Comment by poush** [comment]
@shubham-padia Agree, we can keep it nullable in the database for now. In the follow up issue we can plan for the migration solution.

File: app/models/ticket_holder.py [hunk 1]
```diff
@@ -16,7 +16,7 @@ class TicketHolder(db.Model):
 
     id = db.Column(db.Integer, primary_key=True)
     firstname = db.Column(db.String, nullable=False)
-    lastname = db.Column(db.String)
+    lastname = db.Column(db.String, nullable=False)
```
Comment (poush) [inline app/models/ticket_holder.py]: @mayank8318 revert this change and remove migration file as well.

Comment (mayank8318) [reply app/models/ticket_holder.py]: @poush Ok ☺ I will make the changes by Monday once my exams get over. Is that okay?

**Comment by mayank8318** [comment]
@poush I have updated the PR, please review :)
